### ISSUE TITLE
Homing Key Version

### ISSUE THREAD

**Issue by IronArthur** [levpopov/LPX#2]
are you going to create a version for the homing keys?

**Comment by aranwe** [comment]
If we have the #3|source files and the #1|license permits, I'd be happy to add homing bump. :)

**Comment by IronArthur** [comment]
There is a step file on the project so you already have a source file.

**Comment by aranwe** [comment]
@IronArthur Right, haven't noticed that -> though I had in mind providing something that retains the design features like f3d, ipt, sldprt etc. :) 

Anyway as promised #4 -> @levpopov feel free to merge or delete the PR.

If anyone prints that show the results :) I am out of filament atm.

My contribution is licensed under any CC license that any end-user needs.
Thank you & cheers!

**Comment by IronArthur** [comment]
I forgot to answer, i´ve printed the key and it printed fine, but the notch is too small even printing at 0.1layer. 
when i put my finger on it i can tell there is something there, but i wouldn´t notice it on regular typing.

**Comment by IronArthur** [comment]
> @IronArthur Right, haven't noticed that -> though I had in mind providing something that retains the design features like f3d, ipt, sldprt etc. :)
> 
> Anyway as promised #4 -> @levpopov feel free to merge or delete the PR.
> 
> If anyone prints that show the results :) I am out of filament atm.
> 
> My contribution is licensed under any CC license that any end-user needs. Thank you & cheers!

I forgot to answer, i´ve printed the key and it printed fine, but the notch is too small even printing at 0.1layer.
when i put my finger on it i can tell there is something there, but i wouldn´t notice it on regular typing.
